metricflow/test/integration/test_cases/itest_metrics.yaml

```diff
@@ -1043,3 +1043,64 @@ integration_test:
     ) b
     ON a.metric_time__week = b.metric_time__week
       AND a.metric_time__month = b.metric_time__month
+---
+integration_test:
+  name: simple_query_with_date_part
+  description: Test query using date_part
+  model: SIMPLE_MODEL
+  metrics: ["bookings"]
+  group_by_objs: [{"name": "metric_time", "date_part": "year"}]
+  check_query: |
+    SELECT
+      SUM(1) AS bookings
```

Comment:
Question - what happens with distinct metrics? What should happen?

I'm assuming it'll replace the SUM(1) here with a COUNT(DISTINCT guest_id) or whatever, which seems right, but it might be nice to double check this.

Reply:
I'll add a test case!